bokeh/models/sources.py

```diff
@@ -254,6 +257,91 @@ def _check_column_lengths(self):
         if len(lengths) > 1:
             return str(self)
 
+
+class GeoJSONDataSource(ColumnDataSource):
+    geojson = String(help="""
+    A string of GeoJSON that contains features for plotting. Currently GeoJSONDataSource can
```

Comment:
Should define a `GeoJSONProperty` that subclasses `String` and does validation that the string is correct `GeoJSON`. Also if we currently only support subset, that is the place to check and warn about "extra" things that we don't yet support.